function/src/main/java/io/micronaut/function/executor/FunctionInitializer.java

```diff
@@ -96,7 +95,6 @@ public void close() {
      *
      * @param args The arguments passed to main
      * @param supplier The function that executes this function
-     * @throws IOException If an error occurs
```

Comment:
Otherwise, Javadoc complains of a throw javadoc for an exception not being thrown.